kickstarts/partials/packages/excludes.ks.erb

```diff
@@ -9,15 +9,23 @@
 -ipw2200-firmware
 -ivtv-firmware
 -iwl100-firmware
+-iwl105-firmware
+-iwl135-firmware
 -iwl1000-firmware
+-iwl2000-firmware
+-iwl2030-firmware
+-iwl3160-firmware
 -iwl3945-firmware
 -iwl4965-firmware
 -iwl5000-firmware
 -iwl5150-firmware
 -iwl6000-firmware
 -iwl6000g2a-firmware
+-iwl6000g2b-firmware
 -iwl6050-firmware
-#kernel-firmware
+-iwl7260-firmware
+-iwl7265-firmware
+-kernel-firmware
```

Comment:
I'm ok with this, but I have even less knowledge of these firmware, so I'm cautious about changing them.  I'm ok with changing this but since you dug into this a bit, if there was a missing firmware do you have any idea what step would fail in the build process?